### PR TITLE
Fix data points config edge case for ComboCharts: column and non-colu…

### PR DESCRIPTION
**Pull request by fiedlr**
…mn with no measures

JIRA: ONE-4673

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by fiedlr** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1149/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2753/ : SUCCESS in 9m 38s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1150/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/399/ : SUCCESS in 21m 07s

File: libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/comboChart/PluggableComboChart.tsx [hunk 1]
```diff
@@ -242,10 +242,17 @@ export class PluggableComboChart extends PluggableBaseChart {
...
     }
 
-    private isDataPointsControlDisabled(): boolean {
+    private isDataPointsControlDisabled(insight: IInsightDefinition): boolean {
+        const measureBucketsOfNonColumnCharts = [
+            [this.primaryChartType, BucketNames.MEASURES],
+            [this.secondaryChartType, BucketNames.SECONDARY_MEASURES],
+        ]
+            .filter((chartType) => chartType[0] !== VisualizationTypes.COLUMN)
+            .map((chartType) => insightBuckets(insight, chartType[1]));
```
Comment (no23reason) [inline libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/comboChart/PluggableComboChart.tsx:251]: This argument name is a bit misleading. How about

```ts
.filter(([chartType]) => chartType !== VisualizationTypes.COLUMN)
.map(([, bucketId]) => insightBuckets(insight, bucketId));
```

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/429/ : SUCCESS in 2m 00s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/687/ : SUCCESS in 22s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/2758/ : SUCCESS in 9m 39s

**Pull request merged**